python/dataproc_templates/jdbc/jdbc_to_bigquery.py

```diff
@@ -180,35 +180,50 @@ def run(self, spark: SparkSession, args: Dict[str, Any]) -> None:
         )
 
         # Read
-        input_data: DataFrame
+        input_data: DataFrame = None  # Initialize input_data
 
         partition_parameters = str(input_jdbc_partitioncolumn) + str(input_jdbc_lowerbound) + str(input_jdbc_upperbound)
         if ((partition_parameters != "") & ((input_jdbc_partitioncolumn == "") | (input_jdbc_lowerbound == "") | (input_jdbc_upperbound == ""))):
             logger.error("Set all the sql partitioning parameters together-jdbctogcs.input.partitioncolumn,jdbctogcs.input.lowerbound,jdbctogcs.input.upperbound. Refer to README.md for more instructions.")
             exit (1)
 
-        properties = {constants.JDBC_URL: input_jdbc_url,
-                      constants.JDBC_DRIVER: input_jdbc_driver,
-                      constants.JDBC_TABLE: input_jdbc_table,
-                      constants.JDBC_NUMPARTITIONS: jdbc_numpartitions,
-                      constants.JDBC_FETCHSIZE: input_jdbc_fetchsize}
-        if input_jdbc_sessioninitstatement:
-            properties[constants.JDBC_SESSIONINITSTATEMENT] = input_jdbc_sessioninitstatement
-        if partition_parameters:
-            properties.update({constants.JDBC_PARTITIONCOLUMN: input_jdbc_partitioncolumn,
-                               constants.JDBC_LOWERBOUND: input_jdbc_lowerbound,
-                               constants.JDBC_UPPERBOUND: input_jdbc_upperbound})
-
-        input_data = spark.read \
-            .format(constants.FORMAT_JDBC) \
-            .options(**properties) \
-            .load()
-
-        # Write
-        input_data.write \
-                .format(constants.FORMAT_BIGQUERY) \
-                .option(constants.TABLE, big_query_dataset + "." + big_query_table) \
-                .option(constants.GCS_BQ_TEMP_BUCKET, bq_temp_bucket) \
-                .mode(output_mode) \
-                .save()
+        source_tables_list = input_jdbc_table.split(";")
+        target_tables_list = big_query_table.split(";")
```

Comment:
instead of list can we accept dictionary for source:target.
Keep these 2 variables same to avoid breaking change but add another variable which accepts a key:value of source:target mapping so users do not have to worry about order

Reply:
Actually @surjits254 if we would do that, then we need to change our older pattern of accepting the input parameters. Below mentioned are the two older parameters :-

--jdbc.bigquery.input.table='test.demo' 
--jdbc.bigquery.output.table='jdbctobq'

Thus, without changing any of the older support we need to come up with some solution. Hence, figured out this way. Do you think so we should change the older format to dictionary ?